Unreal/CarlaUE4/Plugins/Carla/Source/Carla/Vehicle/WheeledVehicleAIController.cpp

```diff
@@ -109,7 +109,10 @@ void AWheeledVehicleAIController::OnPossess(APawn *aPawn)
   Vehicle = Cast<ACarlaWheeledVehicle>(aPawn);
   check(Vehicle != nullptr);
   MaximumSteerAngle = Vehicle->GetMaximumSteerAngle();
-  check(MaximumSteerAngle > 0.0f);
+  if (MaximumSteerAngle > 0.0f) {
```

Comment:
Note that in CarlaUE5 `ACarlaWheeledVehicle::GetMaximumSteerAngle()` is always 0, so `MaximumSteerAngle > 0.0f` is always false.